Automattic/jetpack
# webpack-config: Fork calypso-build's mini-css-with-rtl plugin

#### Changes proposed in this Pull Request:

Fork calypso-build's mini-css-with-rtl plugin.
And clean it up with some better formatting, better input validation,
and a better way to override the default "isRTL" expression.

Also, split the "RtlCssPlugins" into two now that StandardPlugins looks
to be the most common way of getting plugins. That makes it more
straightforward to configure the two plugins.

#### Jetpack product discussion


Tangentially related to p9dueE-3MG-p2, in that this mini-plugin would be the last thing left depending on calypso-build so forking it lets us drop the dep on calypso-build and so avoid having to include all its deps.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Probably the only thing affected would be Jetpack's Instant Search module. See if it still works.
* Or try this simple project:
  * Download and unzip [test.zip](https://github.com/Automattic/jetpack/files/7445097/test.zip)
  * Adjust the path to webpack-config in package.json if necessary.
  * `pnpm install && pnpx webpack`
  * Load index.html and index-rtl.html, see if both work right.
